app/assets/stylesheets/application/contributors.scss

```diff
@@ -1,9 +1,49 @@
-.hideContent {
-  display: none;
-}
+.contributorModalBody {
+	display: flex;
+	flex-direction: row;
+
+	.contributorModalBodyPhoto {
+    flex: 0 1 30%;
+    @media screen and (max-width: $small) {
+      flex: 0 1 25%;
+      .contributorModalAvatarMedium img.Avatar__image___2cXeU {
+        width: 100px;
+        height: 100px;
+      }
+    }
+  }
+
+	.contributorModalBodyCopy {
+    margin-left: $size-18;
+    
+    .contributorModalHeader {
+      display: flex;
+      align-items: center;
+
+      h4.contributorModalName {
+        font-size: $size-36;
+        color: $mulberry;
+        line-height: $size-42;
+        font-weight: 400;
+        margin: $size-24 0 $size-10 0;
+        @media screen and (max-width: $small) {
+          margin-top: 0;
+        }
```

Comment:
It would be great if we can add in `text-align: center;` for smaller breakpoints.